tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -113,8 +110,9 @@ def _fit(self, X_train, y_train, time_limit=None, **kwargs):
             self.model = self.model.fit(X_train, y_train)
             if (i == 0) and (len(n_estimator_increments) > 1):
                 time_elapsed = time.time() - time_train_start
-
-                model_size_bytes = sys.getsizeof(pickle.dumps(self.model))
+                model_size_bytes = 0
+                for estimator in self.model.estimators_:  # Uses far less memory than pickling the entire forest at once
+                    model_size_bytes += sys.getsizeof(pickle.dumps(estimator))
```

Comment:
Do we need to get through all of the estimators? One of the option take just 30-ish trees and average them out to get the projection.

Reply:
calculation based on sample, only at most 4 trees are in this loop